src/utils/style-sheet.ts

```diff
@@ -83,7 +80,7 @@ export default class CustomStyleSheet {
         insertRule: (rule: string) => {
           // Enough 'spec compliance' to be able to extract the rules later
           // in other words, just the cssText field
-          (this.sheet!.cssRules as SSCSSRule[]).push({ cssText: rule })
+          ;(this.sheet!.cssRules as SSCSSRule[]).push({ cssText: rule })
```

Comment:
This is kind of weird?

Reply:
@mshwery I agree so I looked into it. 

The `;` makes sure that this
```ts
// something that ends with )
foo()
// something that begins with ( like casting to a type
(bar as baz).foo()
```

doesn't get compiled to
```ts
foo()(bar as baz).foo
```
aka it prevents being parsed as a closure/function call/....
I've seen it happening somewhere else too, it's something prettier/typescript adds to denote a new line / new expression